autoemulate/experimental/calibration/bayes.py

```diff
@@ -63,35 +73,49 @@ def __init__(  # noqa: PLR0913
         self.emulator = emulator
         self.emulator.device = self.device
         self.output_names = list(observations.keys())
+        self.logger, self.progress_bar = get_configured_logger(log_level)
+        self.logger.info(
+            "Initializing BayesianCalibration with parameters: %s", calibration_params
```

Comment:
In case `calibration_params is None`

```suggestion
            "Initializing BayesianCalibration with parameters: %s", self.calibration_params
```